### PR TITLE
Display limited number of comments

### PR DESCRIPTION
**Pull request by shandyk18**
The main page now displays a hardcoded (2) number of comments on the page.

### CODE REVIEW

**Comment by shandyk18** [comment]
I got my branches and merges mixed up, so my master isn't up to speed with my other branches. Other commits are appearing because they aren't on master yet. The very last commit are the changes I actually made.

File: portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java [hunk 1]
```diff
@@ -34,19 +34,26 @@
 public class CommentsServlet extends HttpServlet {
 
   private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-  
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Task");
```
Comment (wujingya) [inline portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java:40]: You should be able to use setLimit(maxComments) in the query, then you don't need the counter.

Comment (shandyk18) [reply portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java:40]: I'm confused on the method. Should I use it like query.setLimit(maxComments)? It's not listed as a valid method,

Comment (wujingya) [reply portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java:40]: Here's an example: https://cloud.google.com/datastore/docs/concepts/queries#cursors_limits_and_offsets

Comment (wujingya) [reply portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java:40]: As we discussed, this documentation is not the right one, let's keep the counter approach for now

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -79,6 +79,8 @@ <h3>Overview</h3>
           </div>
           <div class="container">
             <form action="/comments" method="POST">
+              <textarea name="max-num"></textarea>
+              <br>
```
Comment (wujingya) [inline portfolio/src/main/webapp/index.html:83]: Also as we discussed, this shouldn't be part of this form. Let's leave out the frontend change in this PR and only focus on the server changes, you can make another PR to connect it up.
